readme.md

```diff
@@ -246,6 +254,7 @@ Contains libraries that enable interactions with the Device Proviosining service
 
 Contains libraries that provide modeling and JSON serialization capabilities on top of the raw messaging library.
 
+
```

Comment:
Nit - is the extra line needed here or just some accident of editing?  We don't seem to be consistent in rest of readme.md (most of time we have a single line but other times there's multiple).

Reply:
That was by accident. Removed.